hail/src/main/scala/is/hail/expr/ir/TableIR.scala

```diff
@@ -1508,7 +1508,7 @@ case class TableMapRows(child: TableIR, newRow: IR) extends TableIR {
         FastIndexedSeq(classInfo[Region], LongInfo, LongInfo), LongInfo,
         Coalesce(FastIndexedSeq(
           extracted.postAggIR,
-            Die("Internal error: TableMapRows: row expression missing", extracted.postAggIR.typ))))
+            Die("Internal error: TableMapRows: row expression missing", extracted.postAggIR.typ, -1))))
```

Comment:
You added the constructor that sets the errorId to -1 implicitly; did you not want to use that here (and several other places)? That would keep the -1 magic number hard coded in fewer places.

Reply:
Ah, I originally added the numeric argument everywhere manually because I didn't want to miss anywhere where I had to copy the argument along. I can go back now and remove the extras.